src/routes/OrderConfirmationPage.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { SmileOutlined } from '@ant-design/icons';
+import { Button, Result, Layout } from 'antd';
+import {useLocation, useNavigate} from "react-router-dom";
+
+
+const OrderConfirmationPage = () => {
+    const navigate = useNavigate();
+    const {state} = useLocation();
+    const orderId = state.orderId;
+    const allOrders = state.allOrders;
+
+    const onClickOrderMore = () => {
+        navigate('/user-home/user-home', {state: {allOrders: allOrders}});
```

Comment:
`/user-home/user-home` is not in the route path and click on `Order More` would take users to an empty page. Please change this to `/user-home`.

Reply:
thanks. good catch, will fix